### DISCUSSION TITLE
Preparing for v5.2

### DISCUSSION THREAD

**Discussion by axelboc** [axelboc/anki-ultimate-geography#613] [General]
Woah, can't believe it's been a year and a half since the last release already... I might have a bit of time in a week or so to prepare a new release, so I thought I'd give you all a heads up.

<details>
<summary>Here is the <a href="https://github.com/anki-geo/ultimate-geography/compare/v5.1...master">list of changes</a> since v5.1 for reference:</summary><br>

* Update Spanish and Chinese deck descriptions by @aplaice in https://github.com/anki-geo/ultimate-geography/pull/567
* Expand SADR country info mentioning alternative names by @aplaice in https://github.com/anki-geo/ultimate-geography/pull/570
* issue-573 fix type on Wallis en Futuna by @reinpeter in https://github.com/anki-geo/ultimate-geography/pull/575
* issue-574 fix spelling error on koniNkrijk by @reinpeter in https://github.com/anki-geo/ultimate-geography/pull/576
* Make Astana capital again by @KentGrigo in https://github.com/anki-geo/ultimate-geography/pull/571
* Resize flag of Indonesia and add viewBox to flag of Eswatini by @aplaice in https://github.com/anki-geo/ultimate-geography/pull/578
* Fix minor issues in Astana capital info translations by @aplaice in https://github.com/anki-geo/ultimate-geography/pull/579
* Vectorize flags by @KentGrigo in https://github.com/anki-geo/ultimate-geography/pull/577
* Add viewBox, set height to 250 and adjust width proportionally by @aplaice in https://github.com/anki-geo/ultimate-geography/pull/580
* Add extra info for Sea of Galilee in dutch by @reinpeter in https://github.com/anki-geo/ultimate-geography/pull/582
* Add alternative for capital of China by @reinpeter in https://github.com/anki-geo/ultimate-geography/pull/585
* Add alternatives for the pacific in Dutch by @reinpeter in https://github.com/anki-geo/ultimate-geography/pull/584
* Fix capitalization of Eswatini by @reinpeter in https://github.com/anki-geo/ultimate-geography/pull/586
* Differentiate better between state and region of micronesia by @reinpeter in https://github.com/anki-geo/ultimate-geography/pull/583
* Small changes / additions (T&C Eilanden, Malediven, Oostzee) by @reinpeter in https://github.com/anki-geo/ultimate-geography/pull/588
* Vectorize flags by @KentGrigo in https://github.com/anki-geo/ultimate-geography/pull/587
* [SV] Change from Vitryssland to Belarus  by @melissis in https://github.com/anki-geo/ultimate-geography/pull/599
* Dutch translation corrections by @mhvis in https://github.com/anki-geo/ultimate-geography/pull/601
* Fix: re-lock dependencies to fix `ValueError` during install by @samm81 in https://github.com/anki-geo/ultimate-geography/pull/607
* Update name of Kaliningrad Oblast in Polish by @aplaice in https://github.com/anki-geo/ultimate-geography/pull/609
* Treat Spanish and Chinese decks as distinct "objects" by @aplaice in https://github.com/anki-geo/ultimate-geography/pull/566

</details>

If you'd like to get something else in before the release, **please go ahead!** I've set the [v5.2 milestone](https://github.com/anki-geo/ultimate-geography/milestone/10) on a few issues that I think would be worth including. If you start working on one, don't forget to assign yourself to it so I know something is happening and I should wait before releasing v5.2 (and also to avoid doubling up on work of course).

**Comment by axelboc** [comment]
The draft release notes for v5.2 are [finally ready](https://github.com/anki-geo/ultimate-geography/releases)! 🎉

Hopefully I haven't missed anything. @aplaice, @ohare93, please feel free to fix/improve/amend them directly. Everyone else with read access, don't hesitate to suggest changes here. 😊

**Reply by horwitz** [reply]
I'm not seeing the draft release notes for 5.2 (nor any instance of the string "5.2") at [the link above](https://github.com/anki-geo/ultimate-geography/releases)—where exactly should I be looking?

I (initially) see this when following that link:
<img width="1442" alt="image" src="https://github.com/anki-geo/ultimate-geography/assets/5925829/424cc741-f259-4e67-a4e1-30ec72b2a4eb">

**Reply by aplaice** [reply]
Thanks very much for preparing this, with very clear, nested summaries of the changes!

I _think_ that everything is fine. (Haven't tested import _yet_.)

<hr/>

Unfortunately, GitHub's settings for viewing draft releases are not exactly very logical or intuitive:

https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization

![image](https://github.com/anki-geo/ultimate-geography/assets/16461345/9376c4c9-a048-450f-bf9f-d78f62007869)

so people with read access (such as @horwitz) won't be able to see them...

**Reply by axelboc** [reply]
Would you like write access @horwitz ? Any objection @aplaice ?

**Reply by aplaice** [reply]
No objections!

**Reply by horwitz** [reply]
sounds good to me (if only to see the release notes draft!)—thanks :D

**Reply by axelboc** [reply]
Done 😊

**Reply by horwitz** [reply]
looks great! I made exactly one change—adding a comma just after "2024" in "January 1, 2024 and"

FWIW, once 5.2 is released and the related discussion exists, consider pinning that discussion (and maybe unpinning the ["v5.0 released!"](https://github.com/anki-geo/ultimate-geography/discussions/475) discussion)

**Comment by aplaice** [comment]
Unfortunately, I've only now had time to test the draft release (I was ill).  AFAICT the only thing that we missed is that for Chinese and Spanish, due to the translated deck descriptions, a new "Ultimate Geography [XX]" deck will be created.  I've updated the Upgrading Wiki page! (I hope it makes sense.)

BTW thanks for catching the decreased number of sovereign states!
